src/__tests__/operate.test.js

```diff
@@ -0,0 +1,20 @@
+import operate from '../logic/operate';
+
+describe('testing operations', () => {
+
+  test('test sum', () => {
+    expect(operate(10, 5, '+')).toBe('15');
+  });
+  test('test minus', () => {
+    expect(operate(10, 5, '-')).toBe('5');
+  });
+  test('test divide', () => {
+    expect(operate(10, 5, '÷')).toBe('2');
+  });
+  test('test multiply', () => {
+    expect(operate(10, 5, 'x')).toBe('50');
+  });
+  test('test modules', () => {
+    expect(operate(10, 3, '%')).toBe('1');
+  });
+});
```

Comment:
- [ ] Good job here, however, let us add more tests to move forward.

- Should throw an error if given a wrong number
- Should give a 0 error if given a wrong operator

Reply:
Added test that checks for correct numbers and operators